requirements-development.txt

```diff
@@ -9,6 +9,7 @@ pylint-django
 requests
 mypy==0.560
 coverage
+mock==2.0.0
```

Comment:
Versions should not be hard-coded in the `.txt` file unless you know with certainty that no other version works.

By the way, please also fix the `mypy==0.560` line above. I thought we fixed it in #11?